data/hai/unfettered side missions.txt

```diff
@@ -95,7 +95,11 @@ mission "Hai Legacy 1"
 					decline
 			label first
 			`As you're browsing the shipyard, you are discreetly approached by a hooded figure. When you stop to see what they want to say, they grab and pull on your arm to follow them. "Keep walking," the stranger hisses. After pointing at and commenting on random ships, as if to mimic the actions of a regular passerby, they address you again.`
-			`	"My associates and I have a lucrative business proposition for you. If you are interested, come find us at these coordinates." Without waiting for a response, they fold a piece of paper into your hand before quickly fading into the crowd. It lists only a location, and if you cross-reference it with the local maps, it appears to point towards a beach on the other side of the planet.`
+			`	"My associates and I have a lucrative business proposition for you. It will require your undivided attention, so you would do well to finish any ongoing business first. If you are interested, come find us at these coordinates."`
+			`	Glancing over at your fleet on the nearby landing pad, they add, "It's a solitary affair -- make sure you can park most of your ships."`
```

Comment:
I don't think this works.
From the spaceport description of Hai-home:
https://github.com/endless-sky/endless-sky/blob/20bcaae66d2b7b0f0217ec6f13977d9c40b3fe79/data/map%20planets.txt#L2373
Maybe if I were browsing the shipyard from some sort of computer terminal in the hangar bay where I parked my ships, but then why would the fellow tell me to "keep walking" or point to ships in the shipyard, and why would there be a crowd of passers by?
How about:
```suggestion
			`	"We note you operate multiple ships, however, this is a solitary affair; make sure you can park most of your ships."`
```
Or similar.